### DISCUSSION TITLE
Guidance to create a seven segments digit recognizer

### DISCUSSION THREAD

**Discussion by omr-dz** [PaddlePaddle/PaddleOCR#10277] [Q&A]
I am trying to write an Android app to recognize seven-segments numbers in LCD displays (from a camera capture). Compared to other OCR tasks, this looks a relatively easy one and, yet, as a newcomer to the Computer Vision field, I don't seem to be able to obtain a good quality solution.

I have tried a couple of classic CV approaches to no avail (I have run into issues with the variations of types of digits, when the displays show ghost segments, and low light conditions) . I don't have ML background, but it seems to me that with transfer learning, I should be able to leverage one of the existing OCR models to recognize seven-segments digits. Can anyone give me general pointers on how could I do this with PaddleOCR?

Regards!

Omar

**Comment by enggabrielhabib** [comment]
Train your own dataset.
you may have improved results to read meters LCD
